omar-common-properties.gradle

```
@@ -156,6 +156,34 @@ if (hasJar == null) {
 
 version "${buildVersion}-${buildVersionTag}"
 
+task logInOpenshift() {
+    def login = false
+    def count = 0
+    def numRetries = 3
+    def sleepMillis = 1000
+
+    while (!login && count < numRetries) {
+        try {
+            exec {
+                commandLine 'oc', 'login', '--insecure-skip-tls-verify', openShiftUrl,
```

shouldn't need --insecure-skip-tls-verify
```suggestion
                commandLine 'oc', 'login', openShiftUrl,
```